Guides/Migrating.md

```diff
@@ -40,7 +40,15 @@ Also update dependencies version to latest if required.
 
 And run pod install. If successful, running and testing will work correctly. 
 
-#### 9. In example project, add the newly added destination by setting its relative path.
+#### 9. For tests to compile successfully from command line, code signing for the test targets must be set. (It's enough that it's a personal team, Apple ID does not need to have purchased developer program)
+
+#### 10. It may happen that the pod install has created extra schemes in workspace, simply delete them, but (choose Remove reference) upon delete confirmation.
+
+#### 11. It may also happen that the test targets have not been created, in that case, select scheme -> edit scheme ->  test -> +  and it will automatically create a test scheme.
+
+#### 12. It may also happen that after pod install, all pods are added as targets, in that case, simply select "Manage Schemes" and uncheck all the unrequited targets. (Extra step that will not affect any of the project functionality)
```

Comment:
I'm not entirely sure what you mean by the `unerquited targets`?  Here is what my manage schemes looks like for this PR. Can you specify?
<img width="913" alt="Screen Shot 2020-01-15 at 9 35 44 AM" src="https://user-images.githubusercontent.com/31782219/72456925-706dc000-377a-11ea-8670-0355494086bf.png">

Reply:
Here because I fixed it. But sometimes, you will see e.g "Analytics", "Expecta", are selected, which is not necessary.
We just need the 3 targets we use only.
OCLint is an aggregated target, to show OCLint errors in Xcode rather than command line. 
I'll add more info on that.